homeassistant/components/rachio.py

```diff
@@ -126,17 +134,23 @@ def setup(hass, config) -> bool:
 
     # Enable component
     hass.data[DOMAIN] = person
+
+    # Load platforms
+    for component in SUPPORTED_DOMAINS:
+        discovery.load_platform(hass, component, DOMAIN, {}, config)
```

Comment:
You can push the time config with `{}` to platform without need store it to `hass.data` the hole runtime.

Reply:
I was trying to figure out how that works! Thanks for the hint; I'll fix this.

Reply:
I think that it's nicer to have it all in 1 location, which is the `person`. That is also how it is going to be done when you migrate to config entries.